azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/validator/V2ConfigurationValidator.java

```diff
@@ -79,7 +90,7 @@ public String validateWebContainer() {
         if (runtime == null) {
             return "Pleas config the <runtime> in pom.xml.";
         }
-        if (runtime.getWebContainer() == null) {
+        if (WebContainer.fromString(runtime.getWebContainerRaw()) == null && !WebContainerUtils.isJavaSeWebContainer(runtime.getWebContainerRaw())) {
             return "The configuration <webContainer> in pom.xml is not correct.";
```

Comment:
The message is not helping: The configuration <webContainer> in pom.xml is not correct.